### PR TITLE
Made typography styles while in Storybook more specific.

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Fixes an issue where some styles meant only for crayons typography Storybook stories were leaking out.

**This does not affect application code, only Storybook stories code.**

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [x] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Some guy slapping on some duct tape onto a huge vat of water that is leaking to stop the leak

### CODE REVIEW

**Comment by nickytonline** [comment]
> wasn't sure how to test this out locally (if at all?) but it makes logical sense to me :)

I probably should have put steps to test. 😉 You run Storybook and just check that the Typography section still renders properly. I think I will need to do a session about Storybook and record it.

**Comment by nickytonline** [comment]
codeclimate is complaining about a ruby controller that wasn't even touched. I'm guessing I may need the @maestromac touch to get this merged. 😉

**Comment by maestromac** [comment]
@nickytonline that was a test coverage inconsistency blimp. I've made a PR to addressed that.

File: docs/.ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.7.0
+2.7.1
```
Comment (nickytonline) [inline docs/.ruby-version:1]: This is coming in another PR, but I added it so my build doesn't fail on Netlify.

**Comment by nickytonline** [comment]
Hmm, looks like the CLA bot is jammed again at @maestromac. Can you override that?

**Comment by vaidehijoshi** [comment]
@nickytonline I triggered it for ya! 😉 If you want to try it yourself, you can do this: `https://cla-assistant.io/check/thepracticaldev/dev.to?pullRequest=your-PR-number-here`

**Comment by nickytonline** [comment]
> @nickytonline I triggered it for ya! 😉 If you want to try it yourself, you can do this: `https://cla-assistant.io/check/thepracticaldev/dev.to?pullRequest=your-PR-number-here`

TIL

**Pull request merged**